### PR TITLE
Referral code in Tequilapi payout endpoint

### PR DESCRIPTION
**Pull request by ignasbernotas**
Closes: #1146 

Implements https://github.com/mysteriumnetwork/api/pull/135 endpoint changes

### CODE REVIEW

File: cmd/commands/cli/command.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ import (
 	"strings"
 
 	"github.com/chzyer/readline"
+	"github.com/urfave/cli"
+
```
Comment (vkuznecovas) [inline cmd/commands/cli/command.go]: redundant whitespace

File: market/mysterium/mysterium_api.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ import (
 	"time"
 
 	log "github.com/cihub/seelog"
+
```
Comment (vkuznecovas) [inline market/mysterium/mysterium_api.go]: redundant whitespace

File: tequilapi/endpoints/payout.go [hunk 3]
```diff
@@ -22,6 +22,7 @@ import (
 	"net/http"
 
 	"github.com/julienschmidt/httprouter"
+
```
Comment (vkuznecovas) [inline tequilapi/endpoints/payout.go]: redundant whitespace

File: tequilapi/endpoints/payout_test.go [hunk 4]
```diff
@@ -25,21 +25,27 @@ import (
 	"testing"
 
 	"github.com/julienschmidt/httprouter"
+	"github.com/stretchr/testify/assert"
+
```
Comment (vkuznecovas) [inline tequilapi/endpoints/payout_test.go]: redundant whitespace

File: cmd/commands/cli/command.go [hunk 5]
```diff
@@ -376,21 +377,26 @@ func (c *cliApp) payout(argsString string) {
...
-		payoutSignature := "payout set <identity> <ethAddress>"
+		payoutSignature := "payout set <identity> <ethAddress> <referralCode>"
 		if len(args) < 2 {
 			info("Please provide identity. You can select one by pressing tab.\n", payoutSignature)
 			return
 		}
 
 		var identity, ethAddress string
-		if len(args) == 3 {
+		if len(args) > 2 {
```
Comment (Waldz) [inline cmd/commands/cli/command.go]: I don't thing this field is required. I dont see why person can't get payout wo refferal

Comment (Waldz) [reply cmd/commands/cli/command.go]: Actually it could be different command also: `refferal set <identity> <referralCode>`

Comment (ignasbernotas) [reply cmd/commands/cli/command.go]: It is optional:)

Comment (Waldz) [reply cmd/commands/cli/command.go]: my mistake

**Pull request merged**